extensions/sql-database-projects/src/tools/netcoreTool.ts

```diff
@@ -66,9 +66,9 @@ export class NetCoreTool {
 	public async showInstallDialog(): Promise<void> {
 		let result;
 		if (this.netCoreInstallState === netCoreInstallState.netCoreNotPresent) {
-			result = await vscode.window.showInformationMessage(NetCoreInstallationConfirmation, UpdateNetCoreLocation, InstallNetCore, DoNotAskAgain);
+			result = await vscode.window.showErrorMessage(NetCoreInstallationConfirmation, UpdateNetCoreLocation, InstallNetCore, DoNotAskAgain);
```

Comment:
It's generally not good practice to have an extension wait on user input since that can go on infinitely - instead you should generally be having the extension kick off the async call without awaiting on it and then make sure that the extension can handle the dotnet version not being there/correct in the code that uses it (or at least display a useful error to the user)

Given that the user can already close the notification without acting on it that doesn't seem like it should be hard to change and will increase overall startup performance (since the extension host won't be waiting on this extension to activate)

Reply:
+1 the extension load should not wait on this. It okay that dotnet doesnt get installed but extension should still load.

Reply:
Agreed. I have updated the code so that the extension load doesn't wait on the error message. Reverted back the error message change (since extension should load without any issues now).